modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/view_apps.jsp

```diff
@@ -23,24 +23,24 @@
 <div id="<portlet:namespace />-app-builder-root">
 	<liferay-portlet:resourceURL copyCurrentRenderParameters="<%= false %>" var="baseResourceURL" />
 
-	<react:component
-		data='<%=
-			HashMapBuilder.<String, Object>put(
-				"appsTabs", request.getAttribute(AppBuilderWebKeys.APPS_TABS)
-			).put(
-				"basePortletURL", String.valueOf(renderResponse.createRenderURL())
-			).put(
-				"baseResourceURL", String.valueOf(baseResourceURL)
-			).put(
-				"defaultDelta", PropsValues.SEARCH_CONTAINER_PAGE_DEFAULT_DELTA
-			).put(
-				"deltaValues", PropsValues.SEARCH_CONTAINER_PAGE_DELTA_VALUES
-			).put(
-				"pathFriendlyURLPublic", PortalUtil.getPathFriendlyURLPublic()
-			).put(
-				"userId", themeDisplay.getUserId()
-			).build()
-		%>'
-		module="js/pages/apps/index.es"
-	/>
+		<react:component
+			module="js/pages/apps/index.es"
+			props='<%=
```

Comment:
Everywhere else `props` are being defined outside the tag call. Don't we have any SF rule for this? We just don't care about it?